src/main/java/eu/darkbot/api/objects/Locatable.java

```diff
@@ -0,0 +1,12 @@
+package eu.darkbot.api.objects;
+
+import eu.darkbot.utils.Location;
+
+public interface Locatable {
+
+    double getX();
+    double getY();
+
+    //maybe not necessary
+    Location getLocation();
```

Comment:
probs not necessary, and could even be implemented as a default method since we have x and y